.drone.yml

```diff
@@ -15,70 +18,107 @@ steps:
-      # Exclude schemas for kubeconform
-      KUBERNETES_KUBECONFORM_OPTIONS: "--ignore-missing-schemas"
-      VALIDATE_TERRAFORM_TFLINT: "false"
-      VALIDATE_TERRAFORM_FMT: "false"
-      VALIDATE_TERRAFORM_TERRASCAN: "false"
+      MISE_DATA_DIR: /mise-data
+      MISE_OVERRIDE_CONFIG_FILENAMES: "mise.ci.toml"
+      GITHUB_TOKEN:
+        from_secret: github_token
+    volumes:
+      - name: mise-cache
+        path: /mise-data
     depends_on:
       - clone
-
-  - name: render
-    image: quay.io/sighup/e2e-testing:1.1.0_0.7.0_3.1.1_1.9.4_1.21.12_3.8.7_4.21.1
-    pull: always
     commands:
-      - kustomize build katalog/cluster-autoscaler/base > cluster-autoscaler.yml
+      - |
+        mise use kustomize@5.6.0
+        eval "$(mise activate bash --shims)"
+      - kustomize build katalog/cluster-autoscaler/base > cluster-autoscaler-base.yml
+      - kustomize build katalog/cluster-autoscaler/v1.29.x > cluster-autoscaler-v1.29.yml
+      - kustomize build katalog/cluster-autoscaler/v1.30.x > cluster-autoscaler-v1.30.yml
+      - kustomize build katalog/cluster-autoscaler/v1.31.x > cluster-autoscaler-v1.31.yml
+      - kustomize build katalog/cluster-autoscaler/v1.32.x > cluster-autoscaler-v1.32.yml
+      - kustomize build katalog/cluster-autoscaler/v1.33.x > cluster-autoscaler-v1.33.yml
       - kustomize build katalog/load-balancer-controller > load-balancer-controller.yml
       - kustomize build katalog/node-termination-handler > node-termination-handler.yml
 
-
-  - name: check-deprecated-apis
+  - &check-deprecated-apis
+    name: check-deprecated-apis
     image: us-docker.pkg.dev/fairwinds-ops/oss/pluto:v5
     pull: always
     depends_on:
       - render
     commands:
       # we use --ignore-deprecations because we don't want the CI to fail when the API has not been removed yet.
-      - /pluto detect cluster-autoscaler.yml --ignore-deprecations --target-versions=k8s=v1.32.0
-      - /pluto detect load-balancer-controller.yml --ignore-deprecations --target-versions=k8s=v1.29.0
-      - /pluto detect node-termination-handler.yml --ignore-deprecations --target-versions=k8s=v1.29.0
+      - /pluto detect $${KUBERNETES_MANIFESTS} --ignore-deprecations --target-versions=k8s=v1.33.0
+    environment:
+      KUBERNETES_MANIFESTS: cluster-autoscaler-base.yml
+
+  - <<: *check-deprecated-apis
+    name: check-deprecated-apis-cluster-autoscaler-v129
+    environment:
+      KUBERNETES_MANIFESTS: cluster-autoscaler-v1.29.yml
+
+  - <<: *check-deprecated-apis
+    name: check-deprecated-apis-cluster-autoscaler-v130
+    environment:
+      KUBERNETES_MANIFESTS: cluster-autoscaler-v1.30.yml
+
+  - <<: *check-deprecated-apis
+    name: check-deprecated-apis-cluster-autoscaler-v131
+    environment:
+      KUBERNETES_MANIFESTS: cluster-autoscaler-v1.31.yml
+
+  - <<: *check-deprecated-apis
+    name: check-deprecated-apis-cluster-autoscaler-v132
+    environment:
+      KUBERNETES_MANIFESTS: cluster-autoscaler-v1.32.yml
```

Comment:
I don't it makes sense to test older versions of the autoscaler against 1.33.

I'd rather test them against the corresponding kubernetes version